run-build-functions.sh

```diff
@@ -306,11 +308,14 @@ install_dependencies() {
   # Leiningen
   if [ -f project.clj ]
   then
-    mkdir -p $NETLIFY_CACHE_DIR/m2
-    ln -nfs $NETLIFY_CACHE_DIR/m2 ~/.m2
+    if [ -d $NETLIFY_CACHE_DIR/.m2 ];
```

Comment:
Please remove the semicolon. It looks like we aren't 100% consistent, but the vast majority do not have semicolons.